packages/dashboard/src/components/dashboard/dashboard.tsx

```diff
@@ -24,13 +26,14 @@ interface SelectedFeature {
     runtimeArguments?: string;
 }
 
-export const Dashboard = memo<IDashboardProps>(({ fetchServerState, changeNodeEnvironmentState }) => {
+export const Dashboard = memo<IDashboardProps>(({ fetchServerState, changeNodeEnvironmentState, fetchGraphData }) => {
     const [serverState, setServerState] = useState<ServerState>({
         featuresWithRunningNodeEnvs: [],
         features: {},
     });
 
     const [selectedFeature, setSelectedFeature] = useState<SelectedFeature>({});
+    const [selectedFeatureGraph, setSelectedFeatureGraph] = useState<GraphData | null>(null);
```

Comment:
if you don't provide a default value, wouldn't it will be undefined?

Reply:
I want to be able to nullify it while I'm fetching new data - it also works will with the catch comment you added, it we fail the request I expect it to be null